apps/src/util/experiments.js

```diff
@@ -26,6 +27,13 @@ experiments.getQueryString_ = function () {
 };
 
 experiments.getStoredExperiments_ = function () {
+  // Get experiments on current user from experiments cookie
+  const experimentsCookie = Cookie.get('_experiments' + window.cookieEnvSuffix);
```

Comment:
what's the reason for the leading `_` in the cookie name?

Reply:
also, better to reuse `environmentSpecificCookieName` here

Reply:
oh, I see that `environmentSpecificCookieName` is dashboard-specific, so that might not work without making `environmentSpecificCookieName` work on pegasus too.

Reply:
The `_` is just there because our other cookies have one, I don't know if it means anything: https://github.com/code-dot-org/code-dot-org/blob/0c77b8e8835b0cb4f2152246f69dd9acd315abef/dashboard/config/initializers/devise.rb#L339-L341

`environmentSpecificCookieName` unfortunately doesn't work on pegasus pages.

Reply:
OK. It could be worth checking explicitly here that `window.cookieEnvSuffix` is defined. Could lead to a weird bug if we are looking for a cookie at `_experiments_undefined`.